sites/partners/cypress/integration/06-user-mangement.spec.ts

```diff
@@ -0,0 +1,47 @@
+describe("User Mangement Tests", () => {
+  it("as admin user, should show all users regadless of jurisdiction", () => {
+    cy.login()
+    cy.visit("/")
+    cy.getByTestId("Users-1").click()
+    const rolesArray = [
+      "Partner",
+      "Partner",
+      "Administrator",
+      "Administrator",
+      "Jurisdictional Admin",
+    ]
+    cy.get(`.ag-center-cols-container [row-id="0"] [col-id="roles"]`)
+      .contains(rolesArray[0])
+      .should("have.text", rolesArray[0])
+    cy.get(`.ag-center-cols-container [row-id="1"] [col-id="roles"]`)
+      .contains(rolesArray[1])
+      .should("have.text", rolesArray[1])
+    cy.get(`.ag-center-cols-container [row-id="2"] [col-id="roles"]`)
+      .contains(rolesArray[2])
+      .should("have.text", rolesArray[2])
+    cy.get(`.ag-center-cols-container [row-id="3"] [col-id="roles"]`)
+      .contains(rolesArray[3])
+      .should("have.text", rolesArray[3])
+    cy.get(`.ag-center-cols-container [row-id="4"] [col-id="roles"]`)
+      .contains(rolesArray[4])
+      .should("have.text", rolesArray[4])
+    cy.signOut()
+  })
+
+  it("as jurisdictional admin user, should only see partners/jurisdictional admins on the same jurisdiction", () => {
+    cy.login("jurisdictionalAdmin")
+    cy.visit("/")
+    cy.getByTestId("Users-1").click()
+    const rolesArray = ["Partner", "Partner", "Jurisdictional Admin"]
+    cy.get(`.ag-center-cols-container [row-id="0"] [col-id="roles"]`)
+      .contains(rolesArray[0])
+      .should("have.text", rolesArray[0])
+    cy.get(`.ag-center-cols-container [row-id="1"] [col-id="roles"]`)
+      .contains(rolesArray[1])
+      .should("have.text", rolesArray[1])
+    cy.get(`.ag-center-cols-container [row-id="2"] [col-id="roles"]`)
```

Comment:
This test isn't 100% testing what it indicates. It is testing that with our current data, the first 3 rows aren't admins, and we see them. This test would pass if we had a fourth row that had an admin. It works for this data set, but if we hypothesize that this functionality breaks in the future on a different data set, and we do show admins on accident, and it's any future row, this test is a false positive. Is there a way we can verify not that we are successfully seeing partners / jurisdictional admins, but explicitly that we are not seeing super admins? In general testing this might make more sense as a unit test of the filtering, rather than a full integration test.